libbeat/processors/add_cloud_metadata/add_cloud_metadata.go

```diff
@@ -31,10 +31,17 @@ const (
 
 	// Google GCE Metadata Service
 	gceMetadataURI = "/computeMetadata/v1/?recursive=true&alt=json"
+
+	// Tencent Clound Metadata Service
+	qcloudMetadataHost          = "metadata.tencentyun.com"
+	qcloudMetadataInstanceIdURI = "/meta-data/instance-id"
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
const qcloudMetadataInstanceIdURI should be qcloudMetadataInstanceIDURI